app/cafe.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None | str:
+        if "vaccine" not in visitor.keys():
```

Comment:
`in` keyword will be looking for `vaccine` in dictionary keys by default

Reply:
I was sure, that this method was implemented for fastening the execution ))
Thanks )